handbook/sales/interviews/index.md

```diff
@@ -10,3 +10,7 @@
 	- CEO/CTO: We ask you about what motivates you to do your best work, and we tell you more about the vision for the company (30 minutes)
 4. We check your references
 5. We make you a job offer
+
+
+## The AE Interview Cookbook
```

Comment:
```suggestion
## The AE interview cookbook
```

Reply:
We usually prefer sentence case for headers per the style guide: https://about.sourcegraph.com/handbook/communication/content_guidelines/style_and_mechanics#capitalization. I know the doc itself uses title case, but might as well start somewhere :)